### DISCUSSION TITLE
SomfyPlus on ESP32 DevKit V1

### DISCUSSION THREAD

**Discussion by yoowim** [HomeSpan/HomeSpan#370] [General]
Using SomfyPlus on an ESP32 DevKit V1, it is not obvious what SPI (VSPI or HSPI) to connect to the radio RFM69.
It took some time to find out, but you will need to connect VSPI (MISO / MOSI / CLK) from the ESP32 DevKit V1 to an RFM69 radio.

**Comment by HomeSpan** [comment]
Thanks for noting that.  I agree that it's often hard to figure out which pins to use on any given chip, especially since there are often multiple instances of similar interfaces (such as VSPI vs HSPI).  I've often relied on the pins_arduino.h files that are included in the Arduino-ESP32 Board Manager folder for each so-called ESP32 "variant" (e.g., Adafruit Feather, Espressif Dev Kit, m5Stick...).

To make it easier to find this info, I'll add a new CLI command to HomeSpan that prints out the pins for common interfaces (SPI, I2C, and Serial) for whatever chip is being used.
